ui/src/app/applications/components/application-summary/application-summary.tsx

```diff
@@ -79,6 +91,11 @@ export const ApplicationSummary = (props: {app: models.Application; updateApp: (
             view: app.spec.destination.namespace,
             edit: (formApi: FormApi) => <FormField formApi={formApi} field='spec.destination.namespace' component={Text} />
         },
+        {
```

Comment:
Please move namespace creation to Sync Options section:

![image](https://user-images.githubusercontent.com/426437/88085585-e80e1480-cb3a-11ea-8c2e-c6a5371709ac.png)